keps/sig-api-machinery/4006-transition-spdy-to-websockets/README.md

```diff
@@ -510,23 +507,6 @@ as the downstream connection within the dual concurrent `io.Copy` proxying gorou
 The upstream connection is the same SPDY connection to the container (through the
 Kubelet and CRI).
 
-### Pre-GA: Kubelet `StreamTranslatorProxy`
```

Comment:
restore this paragraph, keep the kubelet `StreamTranslatorProxy` bits, drop out the container runtime websocket bits

Reply:
I believe I have addressed this. Please let me know what you think.